donkeycar/contrib/gradcam/cli.py

```diff
@@ -0,0 +1,432 @@
+"""
```

Comment:
While the code in this file looks fine, the implementation should be integrated within the current `donkey makemovie` command and not within a separate script. We can extend the argument list for `makemovie` to add all the arguments you specify below.